csharp/src/Google.Protobuf.Test.CodeGen/Google.Protobuf.Test.CodeGen.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net451;netstandard1.0</TargetFrameworks>

```

Comment:
Let's set the LangVersion really low?
According to https://github.com/protocolbuffers/protobuf/tree/master/csharp#supported-platforms ,  C# 3 should be enough to compile the generated code.